src/main/groovy/uk/ac/sanger/scgcf/jira/lims/utils/WorkflowUtils.groovy

```diff
@@ -33,30 +29,60 @@ import uk.ac.sanger.scgcf.jira.lims.configurations.ConfigReader
 class WorkflowUtils {
 
     /**
+     * Link a list of plates to the specified workflow issue and transition them if appropriate.
+     *
+     * @param plateActionParams a <code>PlateActionParameterHolder</code> object holding all the parameters
```

Comment:
Really what we're doing here is adding plate issues to a grouping issue. Every issue type has its own 'workflow' so the name is a bit wrong here? addPlatesToGivenGrouping maybe? Same with removePlates.